"""
예외처리 이해하기1
try~except
"""

try:
    print("안녕하세요")
    print(param)
except:
    print('예외가 발생했습니다!')

"""
프로그램이 실행되는 동안 오류가 발생하면 프로그램이 진행될 수 없는 상태가 될 때 except 상황이라고 함
프로그램에 예외가 발생하더라도 프로그램을 중단시키지 않고 프로그램을 계속 진행시킬 수 있도록 하는 구문이 try~except이다

정의되지않은 변수 param을 화면에 출력하는 코드는 예외를 발생시키고 프로그램을 중단시키기에 except를 발생시킴
"""